Using `add_subdirectory` not working on MacOS

Assume the following folder layout:
```
external/
    VulkanMemoryAllocator-Hpp/
    CMakeLists.txt
```
Initialized using `git submodule update --init --recursive`, so the dependencies of this repository are included as well.

And the following contents of `CMakeLists.txt`:
```
add_subdirectory(VulkanMemoryAllocator-Hpp)
```

Results in:
```
[cmake] CMake Error at external/VulkanMemoryAllocator-Hpp/CMakeLists.txt:14 (find_package):
[cmake]   By not providing "FindVulkanMemoryAllocator.cmake" in CMAKE_MODULE_PATH
[cmake]   this project has asked CMake to find a package configuration file provided
[cmake]   by "VulkanMemoryAllocator", but CMake did not find one.
[cmake] 
[cmake]   Could not find a package configuration file provided by
[cmake]   "VulkanMemoryAllocator" with any of the following names:
[cmake] 
[cmake]     VulkanMemoryAllocatorConfig.cmake
[cmake]     vulkanmemoryallocator-config.cmake
[cmake] 
[cmake]   Add the installation prefix of "VulkanMemoryAllocator" to CMAKE_PREFIX_PATH
[cmake]   or set "VulkanMemoryAllocator_DIR" to a directory containing one of the
[cmake]   above files.  If "VulkanMemoryAllocator" provides a separate development
[cmake]   package or SDK, be sure it has been installed.
```

This is caused by `if(EXISTS VulkanMemoryAllocator/CMakeLists.txt)` being false, while the folder does exist. I don't want to install this project (or VMA) locally in order to build my project, but build it on the fly.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ project(
 
 find_package(Vulkan REQUIRED)
 
-if(EXISTS VulkanMemoryAllocator/CMakeLists.txt)
+if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/VulkanMemoryAllocator/CMakeLists.txt)
     add_subdirectory(VulkanMemoryAllocator)
 else()
     find_package(VulkanMemoryAllocator REQUIRED)

